### PR TITLE
Make clear that the two example templates must be used together

### PR DESCRIPTION
**Pull request by santihernandezc**
**What this PR does / why we need it**:

This is a little enhancement over our current docs for message templating.

We've had escalations about our [example of custom templates](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/message-templating/#custom-template-examples) not working. Customers tried to use them as standalone templates, when in reality they should be use together as one depends on the other.

When the notification message fails to parse, customers think that there's something wrong with the example templates. This can be solved by adding an extra step in the docs implying that these templates are meant to be used together.

**Which issue(s) this PR fixes**:

https://github.com/grafana/grafana/issues/47023

### CODE REVIEW

File: docs/sources/alerting/unified-alerting/message-templating/_index.md [hunk 1]
```diff
@@ -125,6 +125,13 @@ Template to render entire notification message:
 {{ end }}
 ```
 
+After you save these two templates, use `mymessage` in the notification message field:
```
Comment (brendamuir) [inline docs/sources/alerting/unified-alerting/message-templating/_index.md:128]:
```suggestion
After saving these two templates, use `mymessage` in the notification message field:
```

**Comment by jtheory** [comment]
Nice, thanks for this tweak!

@JacobsonMT 's suggestion should also help this page -- can we include that too? 

> Here are a few examples of how to use custom templates.
to
> Here's an example of how to use custom templates.

**Comment by jtheory** [comment]
Nice, thanks for this!

@JacobsonMT 's suggestion also looks like a good one - e.g., also change

> Here are a few examples of how to use custom templates.

to

> Here's an example of how to use custom templates.

File: docs/sources/alerting/unified-alerting/message-templating/_index.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ You can use any of the following built-in template options to embed custom templ
 
 ### Custom template examples
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/message-templating/_index.md:82]:
```suggestion
### Example of a custom template
```

File: docs/sources/alerting/unified-alerting/message-templating/_index.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ You can use any of the following built-in template options to embed custom templ
...
 
-Here are a few examples of how to use custom templates.
+Here's an examples of how to use custom templates.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/message-templating/_index.md:84]:
```suggestion
Here's an example of how to use a custom template. You can also use the default template included in the setup.
```

**Review by achatterjee-grafana — APPROVED** [review]
Added minor copy edits. Otherwise looks good. I added a suggestion to make this a numbered list. Thanks, for investigating and updating the docs!

**Comment by santihernandezc** [comment]
@achatterjee-grafana I just added your suggested changes, let me know if it looks good to you!

**Review by achatterjee-grafana — APPROVED** [review]
Looks good!

**Comment by achatterjee-grafana** [comment]
This should be backported to 8.5.x.

**Pull request merged**